composer.json

```diff
@@ -56,8 +56,8 @@
         "pda/pheanstalk": "^3.1",
         "mikey179/vfsstream": "^1.6",
         "amphp/artax": "^3.0",
-        "phpstan/phpstan": "^0.12",
-        "symplify/easy-coding-standard-prefixed": "^8.1"
+        "phpstan/phpstan": "^0.12 <=0.12.66",
```

Comment:
I'm not really happy with this change.... But with the higher versions I ran into problems with phpstan and ecs which weren't straight forward to fix (PhpStan complaining about amphp code, ecs refusing to run at all)